watchtower/README.md

```diff
@@ -27,7 +27,7 @@ you should see something like this
 
 
 ```
- ✔ Container ap_autoupdater  Started
+ ✔ Container watchtower  Started
```

Comment:
one nit-pick, the docker compose named the container `ap_autoupdater`, so this should be `ap_autoupdater`?